### PR TITLE
fix: users can upload non image files for projects

### PR DESCRIPTION
**Pull request by adrianthedev**

### CODE REVIEW

File: app/frontend/covid/covid.js [hunk 1]
```diff
@@ -166,6 +166,24 @@ const Covid = {
 
     Turbolinks.visit(uri.readable())
   },
+  ensureFileType() {
```
Comment (sradu) [inline app/frontend/covid/covid.js:169]: This should be handled server side, no? Client side doesn't feel safe.

**Comment by adrianthedev** [comment]
I agree. I just didn't have time to work on it these past couple of days.
I thought this temporary solution is ok until I take a deep dive in the `direct-upload` gem to see how it works. Forgot to mention this.

**Comment by adrianthedev** [comment]
I seems that active storage direct upload doesn't support server side validation because the file doesn't go through our app and goes directly to S3.

I'll have to explore different options for client side validation. I recommend you merge this for the moment and create a new issue to refactor this.

Some resources on that:
- https://github.com/rails/rails/pull/35390#issuecomment-592278467
- https://gist.github.com/lorenadl/a1eb26efdf545b4b2b9448086de3961d#gistcomment-3111691

**Comment by sradu** [comment]
OK.

**Pull request merged**